Dockerfile

```diff
@@ -15,9 +15,47 @@ COPY ./ ./
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM alpine:3.8
+
+ARG IPMITOOL_REPO=https://github.com/ipmitool/ipmitool.git
+ARG IPMITOOL_COMMIT=IPMITOOL_1_8_18
+
+# Install dependencies for ipmitool
+WORKDIR /tmp
+RUN apk add --update --upgrade --no-cache --virtual build-deps \
```

Comment:
```
apk add --update --upgrade impitool
```

?

Reply:
the build deps should be remove by this line `apk del build-deps`. Building from source has its trade offs. Installing from `apk` also does (we'd almost certainly want to pin the version). I'm open to hearing both sides if folks are leaning one way or another. I could go either way.

Reply:
I could definitely test out with `apk add`.

I used this approach to maintain parity with PBNJ. @jacobweinstock  was there any reason `apk add` didn’t work for PBNJ?

Reply:
it wasn't that `apk add` didnt work. The 1.8.18 version was release in [2016](https://github.com/ipmitool/ipmitool/tree/IPMITOOL_1_8_18) and there hasnt been a new release since then. There have been many commits since that release. Building from source was done to allow us to get newer versions of ipmitool without a release being cut. CVE fixes, new features, etc. I haven't got around to upgrading the version in PBnJ but that was one of the intentions. Trade offs :)

Reply:
Got it 👍  That makes sense. Since the current version we have tested with is `1.8.18` we could probably go ahead with using `apk` to install it and later bring back the building from code if need be.